code/modules/antagonists/melting/melting_spells.dm

```diff
@@ -0,0 +1,106 @@
+//MELTING ABILITIES//
+
+/obj/effect/proc_holder/spell/targeted/mark
+	name = "Mark Champion"
+	desc = "Selects a single human in our range as our champion. They will carry the disease but not be affected by it, and will spread it to others around him. Upon death, they will revive as a greater minion."
+
+	action_background_icon_state = "bg_hive"
+	action_icon_state = "mindswap"
+
+	school = "transmutation"
+	charge_max = 600
+	clothes_req = FALSE
+	invocation = "GIN'YU CAPAN"
```

Comment:
I assume that you copied this from mind transfer?

Reply:
tee hee